### DISCUSSION TITLE
! LaTeX Error: File `sjtuthesis.cls' not found.

### DISCUSSION THREAD

**Discussion by majestichou** [sjtug/SJTUThesis#983] [提问]
**问题描述**:
使用MiKTex工具编译main.txt报错：! LaTeX Error: File `sjtuthesis.cls' not found.

**系统信息**:

- OS版本(带版本号): Windows 10
- TeX 发行版(带版本号): MiKTeX 4.9

**日志信息**:

! LaTeX Error: File `sjtuthesis.cls' not found.

**Comment by LogCreative** [comment]
duplicate #980
